deploy/test-environments/delete_env.sh

```diff
@@ -1,54 +1,82 @@
 #!/bin/bash
-# Given a prefix, delete all the environments that match that prefix
-# Usage: ./delete_env.sh <prefix>
-# Example: ./delete_env.sh test
-# This will delete all the environments that start with "test"
-# It will ask for confirmation before deleting each environment
-# TF_VAR_ec_api_key environment variable should be set
-# AWS CLI should be installed and configured
-# Terraform CLI should be installed and configured
-
-ENV_PREFIX=$1
-[ -z "$ENV_PREFIX" ] && echo "Please provide an environment prefix to delete" && exit -1
-[ -z "$TF_VAR_ec_api_key" ] && echo "Please set TF_VAR_ec_api_key with an Elastic Cloud API Key" && exit -1
+set -e
+
+function usage() {
+  cat <<EOF
+Usage: $0 -p|--prefix ENV_PREFIX [-n|--ignore-prefix IGNORE_PREFIX] [-i|--interactive true|false]
+Cleanup script to delete environments based on a prefix.
+Required flag: -p|--prefix
+EOF
+}
+
+# Defaults
+INTERACTIVE=true
```

Comment:
I would just default it to false and when `--interactive` is used set it to true. Then you can make sure the justfile always use `-i`

Reply:
I think having a confirmation beforehand as default is safer since deletion is hard to reverse